libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/route/line/model/RouteLineResources.kt

```diff
@@ -62,7 +74,10 @@ class RouteLineResources private constructor(
             "routeLineScaleExpression=$routeLineScaleExpression" +
             "routeCasingLineScaleExpression=$routeCasingLineScaleExpression" +
             "routeTrafficLineScaleExpression=$routeTrafficLineScaleExpression" +
-            "trafficBackfillRoadClasses=$trafficBackfillRoadClasses"
+            "trafficBackfillRoadClasses=$trafficBackfillRoadClasses" +
+            "alternativeRouteLineScaleExpression=$alternativeRouteLineScaleExpression" +
+            "alternativeRouteCasingLineScaleExpression=$alternativeRouteCasingLineScaleExpression" +
+            "alternativeRouteTrafficLineScaleExpression=$alternativeRouteTrafficLineScaleExpression"
```

Comment:
Same thing applies to properties above ☝️ 

```suggestion
            "trafficBackfillRoadClasses=$trafficBackfillRoadClasses, " +
            "alternativeRouteLineScaleExpression=$alternativeRouteLineScaleExpression, " +
            "alternativeRouteCasingLineScaleExpression=$alternativeRouteCasingLineScaleExpression, " +
            "alternativeRouteTrafficLineScaleExpression=$alternativeRouteTrafficLineScaleExpression)"
```